src/utils/Utils.js

```diff
@@ -0,0 +1,25 @@
+export function checkWord(guess, solution) {
+
+    //TODO: if allowed words don't contain guess -> invalid
+    var evaluation = [];
```

Comment:
Even though `var` wouldn't cause any issues here I would again use `let` and `const` as using `var` may become a bad habbit.